### PR TITLE
[bitnami/harbor] updated harbor readme nodeport parameters to match actual values

### PR DESCRIPTION
**Pull request by walker-tom**
**Description of the change**

The `service.nodePorts` values in the README Parameters table didn't match the actual values being set in `values.yaml`. They're set to `nil` in `values.yaml`. This PR brings the README inline with the values.

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/harbor/README.md [hunk 1]
```diff
@@ -101,9 +101,9 @@ The following tables list the configurable parameters of the Harbor chart and th
...
+| `service.nodePorts.http`           | The service nodePort Harbor listens on when serving with HTTP                                                                                                                                                                                                                   | `nil`                  |
+| `service.nodePorts.https`          | The service nodePort Harbor listens on when serving with HTTPS                                                                                                                                                                                                                  | `nil`                  |
+| `service.nodePorts.notaryPort`     | The service nodePort Notary listens on. Only needed when `notary.enabled` is set to `true`                                                                                                                                                                                      | `nil`                  |
```
Comment (dani8art) [inline bitnami/harbor/README.md:106]:
```suggestion
| `service.nodePorts.notary`     | The service nodePort Notary listens on. Only needed when `notary.enabled` is set to `true`                                                                                                                                                                                      | `nil`                  |
```

**Review by dani8art — CHANGES_REQUESTED** [review]
Hi @walker-tom , thanks for the PR

Please, take a look at my comment.

File: bitnami/harbor/README.md (hunk 1 above)
Comment (walker-tom) [reply bitnami/harbor/README.md]: Great spot, have updated the PR

**Review by dani8art — APPROVED** [review]
LGTM

**Pull request merged**